src/app/frontend/secret/list/cardlist.html

```diff
@@ -32,18 +32,18 @@
   </kd-zero-state>
   <kd-resource-card-header-columns ng-show="$ctrl.secretList.listMeta.totalItems">
     <kd-resource-card-header-column size="small"
-                                    grow="4"
+                                    grow="2"
                                     sortable="true"
                                     sort-id="$root.SortableProperties.NAME">
       [[Name|Label 'Name' which appears as a column label in the table of secrets (secret list view).]]
     </kd-resource-card-header-column>
     <kd-resource-card-header-column size="small"
                                     grow="2"
-                                    ng-if="$ctrl.areMultipleNamespacesSelected()">
-      [[Namespace|Label 'Namespace' which appears as a column label in the table of secrets (secret list view).]]
```

Comment:
I dont know.  I run the dashboard and test to switch the namespace, it seems it is needed